lib/index.d.ts

```diff
@@ -831,7 +831,7 @@ declare namespace Stardog {
     export namespace user {
 
         interface User {
-            username: string;
+            name: string;
```

Comment:
This fix is correct, but we're tracking the "wrongness" in [this ticket](https://stardog.atlassian.net/browse/VET-2366). 

I'm wondering if we should also change this for `user.create` in this PR, or just record that this was fixed in the ticket.